client/blocks/reader-post-card/standard.jsx

```diff
@@ -2,38 +2,29 @@
  * External Dependencies
  */
 import React from 'react';
-import { partial } from 'lodash';
+import { get, partial } from 'lodash';
 
 /**
  * Internal Dependencies
  */
 import AutoDirection from 'components/auto-direction';
 import Emojify from 'components/emojify';
-import ReaderFeaturedVideo from 'blocks/reader-featured-video';
-import ReaderFeaturedImage from 'blocks/reader-featured-image';
 import ReaderExcerpt from 'blocks/reader-excerpt';
+import FeaturedAsset from './featured-asset';
 
 const StandardPost = ( { post, children, isDiscover, expandCard, postKey, isExpanded, site } ) => {
-	const canonicalMedia = post.canonical_media;
-	let featuredAsset;
-	if ( ! canonicalMedia ) {
-		featuredAsset = null;
-	} else if ( canonicalMedia.mediaType === 'video' ) {
-		featuredAsset = (
-			<ReaderFeaturedVideo
-				{ ...canonicalMedia }
-				videoEmbed={ canonicalMedia }
-				onThumbnailClick={ partial( expandCard, { postKey, post, site } ) }
-				isExpanded={ isExpanded }
-			/>
-		);
-	} else {
-		featuredAsset = <ReaderFeaturedImage imageUrl={ canonicalMedia.src } href={ post.URL } />;
+	let onVideoThumbnailClick = null;
```

Comment:
I wondered about this! I thought perhaps it was wasted effort creating the `partial()` if it's not a video? I suspect video cards only make up a few % of total cards, so we'd be often creating it and not using it.